engines/python/setup/djl_python/neuron_utils/model_loader.py

```diff
@@ -808,3 +808,115 @@ def partition(self, save_path: str, **kwargs):
         self.load_pipeline(**kwargs)
         self.save_pipeline(save_path)
         return self.pipeline
+
+
+class TNXVllmModelLoader(ModelLoader):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.generation_config = self.generate_generation_config()
+        self.model_config.generation_config = self.generation_config
+        self.neuron_config = self.generate_neuron_config()
+
+    def generate_generation_config(self):
+        generation_config = NeuronGenerationConfig(
+            max_length=self.config.max_model_len,
+            do_sample=True,
+            per_batch_line=True,
+            top_k=[1] * self.config.max_rolling_batch_size,
+            top_p=[1] * self.config.max_rolling_batch_size,
+            temperature=[1] * self.config.max_rolling_batch_size,
+            dynamic=True,
+            global_top_k=256,
+            deterministic=True,
+        )
+        return generation_config
+
+    def generate_neuron_config(self):
+        continuous_batching_config = None
+        if self.config.max_rolling_batch_size > 1:
+            continuous_batching_config = ContinuousBatchingConfig(
+                batch_size_for_shared_caches=self.config.max_rolling_batch_size)
+
+        on_dev_sampling_config = None
+        if self.config.on_device_generation:
+            on_dev_sampling_config = copy.deepcopy(self.model_config.generation_config)
+
+        quant_config = None
+        if self.config.neuron_quant:
+            quant_dtype = 's8'
```

Comment:
Do we want to make these setable - s8 will be naive weight only quant, but they might start brining in fp8 etc. We should be able to set quant and dequant dtypes somewhere for the long term.